.github/workflows/ui-ci.yml

```diff
@@ -33,11 +33,11 @@ jobs:
         env:
           CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
         run: |
-          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./ui/cc-test-reporter
-          chmod +x ./ui/cc-test-reporter
-          ./ui/cc-test-reporter before-build
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter before-build
           if [ -f coverage/lcov.info ]; then
-            ./ui/cc-test-reporter after-build --exit-code $? --coverage-input-type lcov
+            ./cc-test-reporter after-build --exit-code $? --coverage-input-type lcov
```

Comment:
I think we do need /ui

Reply:
@macintushar Both works as we just need to run the cc from where it's downloaded

Reply:
@karthik-sivadas should it be changed or is it okay?

Reply:
It is ok to leave it as it is as we are getting the code climate

Reply:
Understood, merging